guide/xml/portfile-keywords.xml

```diff
@@ -277,5 +277,29 @@
       </listitem>
     </varlistentry>
 
+    <varlistentry>
+      <term>use_xcode</term>
+
+      <listitem>
+        <para>Available in 2.6.0 and later. By default, it is assumed on
+        macOS that ports will not need tools from Xcode.app unless (1)
+        Command Line Tools aren't installed, (2) you are on an old version
+        of Mac OS X that does not support the xcode-select mechanism, or (3)
+        the port uses <code>build.type xcode</code> or includes the
+        <code>xcode</code> PortGroup. If a port needs to use Xcode (i.e.,
+        xcodebuild) in any way, <code>use_xcode yes</code> should be set or
+        the port should be included in the xcode PortGroup. The environment
```

Comment:
This still has the old wording for the PortGroup inclusion.

Reply:
Didn't catch that. Done